tests/microbenchmarks.py

```diff
@@ -14,6 +14,7 @@ def test_small_integers(test_case):
                     x = x + 1
         """), timed=True)
 
+
 def test_booleans(test_case):
     print("Running", "test_booleans")
     test_case.runAsJava(adjust("""
```

Comment:
At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils